addons/scratchr2/profile.css

```diff
@@ -154,3 +154,9 @@
 .portrait [data-control="edit"] {
   border-top-left-radius: 5px;
 }
+
+.box#profile-data .box-head {
```

Comment:
I like this, but it actually does make a change to the bottom left corner of the profile picture when the banner addon is off. Adding `::before` to the end of the selector makes it target the image itself, which isn't there when the banner is off.

`.box` at the beginning is unnecessary.

Reply:
soo you want it to be `::before #profile-data .box-head`?

Reply:
The correct selector is `#profile-data .box-head::before`.

Reply:
okie dokie